kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1231,6 +1231,21 @@ object KyuubiConf {
       .checkValue(_ > 0, "must be positive number")
       .createWithDefault(Duration.ofMinutes(5).toMillis)
 
+  val KUBERNETES_APPLICATION_STATE_FROM_CONTAINER: ConfigEntry[Boolean] =
+    buildConf("kyuubi.kubernetes.application.state.fromContainer")
+      .doc("If set to true then the application state will be retrieved from the container " +
+        "instead of the pod.")
+      .version("1.8.1")
+      .booleanConf
+      .createWithDefault(false)
+
+  val KUBERNETES_APPLICATION_STATE_CONTAINER: OptionalConfigEntry[String] =
+    buildConf("kyuubi.kubernetes.application.state.container")
```

Comment:
I couldn’t recall the details well. how can we configure the container name? does it have a default name? throw pod template?

Reply:
will give a default value `spark-kubernetes-driver`